mentat/conversation.py

```diff
@@ -86,14 +99,13 @@ async def display_token_count(self):
             config.model,
         )
 
+        context_size = self._get_max_tokens()
         if not context_size:
             raise MentatError(
                 f"Context size for {config.model} is not known. Please set"
                 " maximum-context with `/config maximum_context value`."
             )
-        else:
-            self.max_tokens = context_size
-        if tokens + 1000 > context_size:
```

Comment:
Thanks for catching this! I thought I replaces all the token_buffer spots already but I must have missed this!